omod/src/main/webapp/resources/htmlforms/055b-HCTClientCard.xml

```diff
@@ -396,6 +396,13 @@
                         <obs conceptId="5497"/>
                     </td>
                 </tr>
+                <includeIf velocityTest="$fn.globalProperty('ugandaemr.hts.recency') == true">
+                    <tr>
+                        <td colspan="2"><label>HIV Recency Test Result:</label>
+                            <obs conceptId="141520" answerConceptIds="141518,141519,163611,165145,1118" answerLabels="Recent,Long-term,Invalid,Missing,Not done"/>
```

Comment:
@echiteri Replace "Long-term" with "Long Term" and "Not done" with "Not Done" maintaining the title case